### ISSUE TITLE
Nextcloud 15 compatibility

### ISSUE THREAD

**Issue by violoncelloch** [ONLYOFFICE/onlyoffice-nextcloud#37]
Nextcloud 15 has been released today. It would be nice if you could update this app to work on Nextcloud 15! Thank you!

**Comment by JugularBlua** [comment]
i confirm this  App "ONLYOFFICE" cannot be installed because it is not compatible with this version of the server.
on Nextcloud 15.

**Comment by UBTce** [comment]
I confirm the installed app is not compatible with Nextcloud 15.

Could it be that it still actually is compatible but it is just not activated/allowed for NC15? Maybe by changing the info.xml to allow 15 it will just work again?

Just 24 minutes ago the version NC15 was allowed in the info.xml so just before writing I guess the problem was solved? Update and all is good?

See:
    <dependencies>
        <nextcloud min-version="14" max-version="**15**"/>
    </dependencies>

**Comment by UBTce** [comment]
Solved in [v2.1.2](https://github.com/ONLYOFFICE/onlyoffice-nextcloud/releases/tag/v2.1.2) ?

**Comment by UBTce** [comment]
@LinneyS 
I updated to 2.1.2 but it still says:

> This app cannot be installed because the following dependencies are not fulfilled:
> Server version 14 or lower is required.

and I can not enable the app.

**Comment by LinneyS** [comment]
How do you update?
Download the archive or update from official market?
https://apps.nextcloud.com/apps/onlyoffice/releases?platform=15

**Comment by violoncelloch** [comment]
2.1.2 should be compatible with 15 as listed in the nextcloud app  store

**Comment by UBTce** [comment]
I updated from within Nextcloud so in the installed apps section I saw an update avaliable for onlyoffice to v.2.1.2 and installed. It is strange because in the info.xml it states: `<nextcloud min-version="14" max-version="15"/>` but in Nextcloud I see the error that "Server version 14 or lower is required" which does not fit with min-version="14".

**Comment by magenbrot** [comment]
I had to log out and log back in to enable the onlyoffice app after upgrading to v2.1.2

**Comment by UBTce** [comment]
@magenbrot  oh, that was easy (working again)! I was almost about to reboot the whole server. Thank you for the hint!

**Comment by clinkadink** [comment]
I had the same issue when manually installing Nextcloud (apache) and then OnlyOffice (nginx) this evening (on Ubuntu 18.04) on the same server (no docker or snap). I used this command to install the OnlyOffice app:

`sudo git clone https://github.com/ONLYOFFICE/onlyoffice-nextcloud.git onlyoffice`

I received the following message (in the terminal and in Nextcloud settings):

> This app cannot be installed because the following dependencies are not fulfilled:
> Server version 10 or lower is required.

And the "Enable" button was greyed out in the Nextcloud settings page.

I tried removed and tried again, no luck. I saw an above comment saying this has been resolved in 2.1.2, so tried cloning again, but with depth=1 parameter - and the "Enable" button became 'enabled', and now it works fine.

`sudo git clone --depth=1 https://github.com/ONLYOFFICE/onlyoffice-nextcloud.git onlyoffice`

I must say, very impressed with the integration. Many thanks to the op, and the devs for their efforts.
